### ISSUE TITLE
Access to Shared Drives

### ISSUE THREAD

**Issue by lifeboy** [astrada/google-drive-ocamlfuse#672]
Google offers "My Drive" and "Shared Drives".  Is there a way in which we can connect ocamlfuse to shared drives?   Google offers google-drive-stream for windows, but not for Linux.

**Comment by astrada** [comment]
Yes, in the docs they are still referenced as Team Drives but they should work nonetheless: https://github.com/astrada/google-drive-ocamlfuse/wiki/Team-Drives

**Comment by lifeboy** [comment]
Excellent, works well.  I do notice some lag in updating changes to the steam.  It takes sometime 5 minutes before one sees a new file created in a shared drive.  Why would that be?  It should in the browser interface in Google drive right away.

**Comment by ShnitzelKiller** [comment]
This does not work for me. The folder mounted team folder still contains the contents of my personal drive, even though I've set the team drive ID in the config file for that label.

**Comment by nicolopozzato** [comment]
> This does not work for me. The folder mounted team folder still contains the contents of my personal drive, even though I've set the team drive ID in the config file for that label.

I have also the same issue... i have connected correctly because i can see and manage "My Files" but if i unmount, add the team drive id and redo the google-drive-ocamlfuse -label "label" "mount" I see again my files and not the team drive

**Comment by tilenkranjc** [comment]
I had a similar problem and solved it by emptying the `~/.gdfuse/<label>/cache/` folder. After that I mounted it again and it loaded the team drive.

You can also add an option `-cc` when mounting which does the same.

Might be a solution for #578 #567 #477 #401

**Comment by anishjp** [comment]
@astrada If I have more than one folder in the .shared folder, how do I specify the team_drive_id of all of them in the config file? Do I just add all of them separated by a comma?
